test/lint.js

```diff
@@ -138,10 +134,12 @@ const checkFiles = (...files) => {
  * @returns {boolean} Whether there were any errors
  */
 const main = (files) => {
+  let errors;
+
   if (files) {
```

Comment:
One more improvement opportunity:

```js
if (!files) {
  files = ['api', 'browsers', 'css']
}

const errors = checkFiles(...files);
```

Or would it be possible to just use a parameter default value?

```js
const main = (files =  ['api', 'browsers', 'css']) => {
  const errors = checkFiles(...files);
```